src/appleseed/renderer/kernel/rendering/masterrenderer.cpp

```diff
@@ -292,6 +292,9 @@ IRendererController::Status MasterRenderer::render_frame_sequence(
             return IRendererController::AbortRendering;
         }
 
+        if (components.is_backward_light_sampler_used())
```

Comment:
Instead of `is_backward_light_sampler_used()`, couldn't we simply check if `get_backward_light_sampler()` returns `nullptr` or not?